src/main/java/com/senecafoundation/Scene/Choice.java

```diff
@@ -0,0 +1,30 @@
+package com.senecafoundation.Scene;
+
+import javax.persistence.Entity;
+
+@Entity
+public class Choice
+{
+    //Variables
+    private int choiceOptionNumber;
```

Comment:
Did I seriously not give choices IDs?  Bad job me.

Reply:
should I give both the response and choice classes a ---------> private int id; 

and add-------------> this.id = UUID.randomUUID().toString(); to their constructors?